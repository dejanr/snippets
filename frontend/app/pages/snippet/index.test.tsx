import { expect, test } from 'vitest'
import { render } from '@testing-library/react'
import { SnippetPage } from './index'

test('it renders', () => {
  const { container } = render(<SnippetPage snippet={{ id: '1', text: 'text', summary: 'summary' }} />)

  expect(container.firstChild).toMatchSnapshot()
})
