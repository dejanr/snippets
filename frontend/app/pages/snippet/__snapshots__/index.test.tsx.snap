// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`it renders 1`] = `
<div
  class="container mx-auto p-4 max-w-4xl"
>
  <div
    class="space-y-4"
  >
    <h1
      class="text-3xl font-bold text-gray-800"
    >
      Snippet
    </h1>
    <span>
      text
    </span>
    <span>
      summary
    </span>
  </div>
</div>
`;
